lib/ProductOpener/DataQualityFood.pm

```diff
@@ -659,6 +659,9 @@ sub check_nutrition_data ($product_ref) {
 					"en:nutrition-data-prepared-without-category-dried-products-to-be-rehydrated";
 			}
 		}
+		if ((defined $product_ref->{serving_size}) && (($product_ref->{serving_size} =~ /(\d+)/) eq "")) {
```

Comment:
The test is better written this way I think:
- use !~ //   -> indicates it does not match
- check that the field is not an empty string
- check if there's one digit /\d/

```suggestion
		if ((defined $product_ref->{serving_size}) and ($product_ref->{serving_size} ne "") and ($product_ref->{serving_size} !~ /\d/) ) {
```